.gitignore

```diff
@@ -1 +1,161 @@
 *.swp
+# Byte-compiled / optimized / DLL files
+__pycache__/
```

Comment:
Why do we need such an extensive list of things to ignore? Most of this stuff is not related to our repository.